packages/core/src/browser/quick-open/quick-command-service.ts

```diff
@@ -221,8 +221,8 @@ export class CommandQuickOpenItem extends QuickOpenGroupItem {
     }
 
     getIconClass(): string | undefined {
-        const toggleHandler = this.commands.getToggledHandler(this.command.id);
-        if (toggleHandler && toggleHandler.isToggled && toggleHandler.isToggled()) {
+        const toggledHandler = this.commands.getToggledHandler(this.command.id);
+        if (toggledHandler) {
```

Comment:
`this.commands.isToggled(this.command.id)`?

Reply:
I don't really since I find the mechanic a bit weird, but `.getToggledHandler` now returns a handler that is already tested to be toggled.

https://github.com/theia-ide/theia/blob/3d28b101cf924dfee4f83621b857b7f39cab8b51/packages/core/src/common/command.ts#L331-L349

So I think we can skip checking against the command registry, which would call `getToggledHandler` once again anyway.

https://github.com/theia-ide/theia/blob/3d28b101cf924dfee4f83621b857b7f39cab8b51/packages/core/src/common/command.ts#L262-L264